classes/manager.php

```diff
@@ -126,18 +126,18 @@ public static function on_flush(\ExcimerLog $log, float $started): void {
         $stopped = microtime(true);
         $duration = $stopped - $started;
 
+        $reason = 0;
```

Comment:
Maybe have reason none = 0 as const

Reply:
Added now, but since this should NEVER be a reason to store the profile, I've skipped adding a langstring for it, as it should never be stored and thus displayed.